data_management/access_copy_converter/access_copy_converter/main.py

```diff
@@ -125,50 +125,49 @@ def process_consignment(
         f"{len(response['Contents'])} files in consignment {consignment_ref}"
     )
 
-    with conn:
-        for obj in response["Contents"]:
-            key = obj["Key"]
-            file_id = key.split("/")[-1]
-            logger.info(f"Checking file_id: {file_id}")
+    for obj in response["Contents"]:
```

Comment:
this change is needed because we now pass in the connection to the function as opposed to creating one per consignment